app/controllers/catalog_controller.rb

```diff
@@ -56,12 +56,12 @@ class CatalogController < ApplicationController
 
     # exploded_project_tag_ssim indexes all project tag prefixes for hierarchical facet display, whereas
     #   project tag_ssim only indexes whole tags
-    config.add_facet_field 'exploded_project_tag_ssim', label: 'Project', limit: 9999,
-                                                        component: LazyProjectTagFacetComponent,
+    config.add_facet_field 'exploded_project_tag_ssim', label: 'Project', limit: 100_000,
+                                                        component: Blacklight::Hierarchy::FacetFieldListComponent,
                                                         unless: ->(controller, _config, _response) { controller.params[:no_tags] }
```

Comment:
we aren't using the `LazyProjectTagFacetComponent` anymore ?

Reply:
No, that was something I was testing that I forgot to roll back. Good catch.